housekeeping-vacuum-tool/src/main/java/com/hotels/housekeeping/tool/vacuum/VacuumToolApplication.java

```diff
@@ -172,7 +172,7 @@ void vacuumTable(String databaseName, String tableName)
     LOG.debug("Glob path: '{}'", globPath);
     int globDepth = globPath.depth();
 
-    Set<Path> metastorePaths = pathResolver.getMetastorePaths(batchSize, expectedPathCount);
+    Set<Path> metastorePaths = pathResolver.getMetastorePaths(batchSize);
```

Comment:
OK, so I had a look at what this value was being used for previously. From what I could see in `UnpartitionedTablePathResolver` it wasn't being used at all, while in `PartitionedTablePathResolver` it was only being used to initialise the size of a `HashSet`. I'm not convinced this has much/any utility so I'm fine with it being removed. We had a developer on the project in the past who seemed to love this kind of low level "performance/memory management" but I personally don't think it actually did anything useful. I've always advocated using the defaults and only using magic numbers like this when you run into an issue and can prove they help.

Reply:
Sounds good to me, but what would be the default value? Because there doesn't seem to be an option to not give a number. I could change that, of course, because `PartitionIterator` is in hcommon-hive-metastore :)

Reply:
The default value is no value right? `expectedPathCount` was only being used to control the size of the HashSet wasn't it? So that's why you removed it?

Reply:
ohhh, sorry, I thought you meant that I could remove `batchSize` too 😆

Reply:
No ;)